src/test/js/log-karaoke/stages.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
             })
         ;
         // test whether the expand works
-        nodeDetail.clickFirstResultItem(true);
+        nodeDetail.clickFirstResultItem();
```

Comment:
;) actually I was up to remove that true the first time I implement it, but forgot about it ;)